soc/nordic/nrf54l/Kconfig.defconfig

```diff
@@ -46,4 +46,8 @@ config BUILD_OUTPUT_ADJUST_LMA
 
 endif # RISCV
 
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default $(dt_node_int_prop_int,/soc/peripheral@40000000/grtc@e2000,clock-frequency) if NRF_GRTC_TIMER && TRUSTED_EXECUTION_NONSECURE
```

Comment:
This makes an assumption about the GRTC peripheral address, which does not scale well.
Proper way to address this would be to add `dt_nodelabel_int_prop_int` functionality and use it here with `grtc` label.

Reply:
Completely overlooked that function, will update

Reply:
I meant this dt function should be implemented and then used here - it is not present now.